### ISSUE TITLE
BFA can't equal attack performance in vgg11-bn-quan.

### ISSUE THREAD

**Issue by pdh930105** [elliothe/BFA#7]
Hello, I tested BFA attack in this code but can't equal attack performance in _vgg11-bn-quan_, unlike _resnet_.  (N_flips is 11 on paper, but 50 on test this code).

I think that the attack performance gap is caused by using different pre-trained weights.

So, plz uploaded your pre-trained vgg11-bn-quan weight?

**Comment by adnansirajrakin** [comment]
VGG without the Batchnorm model used in the paper can be found in the following link: https://drive.google.com/drive/folders/1vWnL3oz5gycpAgJtxFMpAtPs99lgHveb?usp=sharing. 
It requires 9-16  flips. We do not have the pre-trained model with the batch norm. But With a batch norm, it is fine to achieve a number of bit-flip within the 20-50 range depending on the pre-trained weights.
